content/en/docs/tasks/security/authz-http/index.md

```diff
@@ -32,178 +30,243 @@ the product page, you can see the following sections:
-$ kubectl delete -f @samples/bookinfo/platform/kube/rbac/namespace-policy.yaml@
+$ kubectl delete authorizationpolicy.security.istio.io/deny-all -n istio-system
 {{< /text >}}
 
-## Enforcing Service-level access control
-
-This task shows you how to set up service-level access control using Istio authorization. Before you start, please make sure that:
-
-* You have [enabled Istio authorization](#enabling-istio-authorization).
-* You have [removed namespace-level authorization policy](#cleanup-namespace-level-access-control).
-
-Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`). You should see `"RBAC: access denied"`.
-We will incrementally add access permission to the services in the Bookinfo sample.
-
-### Step 1. allowing access to the `productpage` service
-
-In this step, we will create a policy that allows external requests to access the `productpage` service via Ingress.
-
-Run the following command:
+## Enforce namespace-level access control
 
-{{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enforcing_service_level_access_control_step1_apply.sh" >}}
+With Istio, you can easily setup namespace-level access control. You can configure whether
+workloads from a namespace can access all or a collection of workloads in another namespace.
 
-Once applied, the policy has the following effects:
-
-*   Creates a `ServiceRole` `productpage-viewer` which allows read access to the `productpage` service.
-
-    {{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enforcing_service_level_access_control_step1_productpage_viewer.yaml" >}}
-
-*   Creates a `ServiceRoleBinding` `bind-productpage-viewer` which assigns the `productpage-viewer` role to all
-users and services.
-
-    {{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enforcing_service_level_access_control_step1_bind_productpage_viewer.yaml" >}}
-
-Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`). Now you should see the "Bookinfo Sample"
-page. But there are errors `Error fetching product details` and `Error fetching product reviews` on the page. These errors
-are expected because we have not granted the `productpage` service access to the `details` and `reviews` services. We will fix the errors
-in the following steps.
-
-{{< tip >}}
-There may be some delays due to caching and other propagation overhead.
-{{< /tip >}}
+The Bookinfo sample deploys the `productpage`, `reviews`, `details`, `ratings` {{< gloss "workload" >}}workloads{{< /gloss >}}
+in the `default` namespace. Istio deploys its components, for example `istio-ingressgateway`,
+in the `istio-system` namespace.
 
-### Step 2. allowing access to the `details` and `reviews` services
+The following command creates a `bookinfo-viewer` policy that allows workloads in the `default`
+and `istio-system` namespaces to access workloads in the `default` namespace with `GET` method:
```

Comment:
[nit] It may cause some confusion between the use of `istio-system` as root namespace, and as a namespace that some workload run. It would be best if the setup use different value for root namespace.

Reply:
I think the current installation uses `istio-system` as the default root namespace, should be updated to a separate value.

Reply:
You may avoid include it here for simplicity. 

I also recommend do the workload policy first to grand ingress to access product-page, and showing the result that request to ingress now get to the product page but no content (you still have global deny-all). Then next step is adding namespace policy for all bookinfo apps, and now every thing works again.